.gitignore

```diff
@@ -52,3 +52,6 @@ npm-shrinkwrap.json
 
 # macOS
 .DS_Store
+
+# vscode settings
+.vscode
```

Comment:
this should probably go in a global gitignore rather than in every individual project

Reply:
Some projects I work in do keep some workspace settings in source control.  This is the only one where I need to actively turn off formatting.  I can remove it, if you prefer.  it's just a bit of a pain to always have to remember to turn it off.

Reply:
hmm - i'd expect those projects would explicitly *un*ignore it in their gitignore since so many people globally gitignore vscode settings already. certainly we can keep it in here if it's that important to you.